.github/workflows/build.yaml

```
@@ -140,9 +167,9 @@ jobs:
           PKG_CONFIG_ALLOW_CROSS: "1"
-          PKG_CONFIG_PATH: "/usr/lib/aarch64-linux-gnu/pkgconfig"
-          SASL2_DIR: "/usr/lib/aarch64-linux-gnu"
-          OPENSSL_DIR: "/usr/lib/aarch64-linux-gnu"
-          OPENSSL_ROOT_DIR: "/usr/lib/aarch64-linux-gnu"
+          PKG_CONFIG_PATH: /usr/lib/aarch64-linux-gnu/pkgconfig
+          SASL2_DIR: /usr/lib/aarch64-linux-gnu
+          OPENSSL_DIR: /usr/lib/aarch64-linux-gnu
+          OPENSSL_ROOT_DIR: /usr/lib/aarch64-linux-gnu
           OPENSSL_STATIC: "1"
-          SASL2_STATIC: "0"
+          SASL2_STATIC: "0"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)